lib/course/widgets/lesson_screen.dart

```diff
@@ -83,7 +81,7 @@ class _LessonScreenState extends State<LessonScreen> {
         }
 
         final lesson = update.data;
-        // TODO(marcelgarus): use FancyScaffold when flutter_cached supports slivers
+        final firstContent = lesson.contents.where((c) => c.isVisible).first;
```

Comment:
```suggestion
        final firstContent = lesson.contents.firstWhere((c) => c.isVisible);
```
Also, what if there is no content for this lesson? Throwing an error is probably not the right thing to do…

Reply:
For now we show a button to view this lesson in the browser (and hope the user selects to open this link in a browser and not in our app).
In the future, we should improve this → #236